packages/vue/src/components/atoms/SfOverlay/SfOverlay.vue

```diff
@@ -0,0 +1,3 @@
+<script src="./SfOverlay.js"></script>
+<template lang="html" src="./SfOverlay.html"></template>
+<style lang="scss" src="./SfOverlay.scss"></style>
```

Comment:
you forgot to move scss files to shared package

Reply:
that's not true ù.ù i did that

Reply:
did that here https://github.com/DivanteLtd/storefront-ui/pull/162/commits/4297655700fc955da13dcbc1e8a082f9c143fde3